### ISSUE TITLE
Support Multiplayer

### ISSUE THREAD

**Issue by paulsteele** [paulsteele/Ferngill-Supply-And-Demand#1]

**Comment by paulsteele** [comment]
```
System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at fse.core.patches.ShopMenuPatches.DrawSeedInfo(SpriteBatch b, ShopMenu __instance) in C:\Users\Paul_\projects\Ferngill-Supply-And-Demand\FerngillSimpleEconomy\patches\ShopMenuPatches.cs:line 66
   at StardewValley.Menus.ShopMenu.draw_PatchedBy<paulsteele.fse__spacechase0.BiggerBackpack>(ShopMenu this, SpriteBatch b)
   at StardewModdingAPI.Framework.SModHooks.TryDrawMenu(IClickableMenu menu, Action draw_menu_action) 
 ```
 
 is one reported error. Client opening the shop.

**Comment by paulsteele** [comment]
resolved via 347b96196693ccb9aa366dc820ad99355e4373ec
